data/categories.txt

```diff
@@ -1865,7 +1865,7 @@ amenity-fountain
 en:fountain|U+26F2
 ru:фонтан
 uk:фонтан
-de:Brunnen|Fontäne
+de:Springbrunnen|Fontäne
```

Comment:
A "Brunnen" or well is the opposite of a fountain. And wouldn't searching for "Brunnen" also match on "Spring**brunnen**?